UnitedIncome/components
# [APP-5041] typescript QuestionCard component

### Description
Converted the `QuestionCard` component and its spec file to typescript.

### Testing Instructions
Check that there are no visual or behavior changes in the storybook

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5041)
